config/database.yml

```diff
@@ -10,15 +10,25 @@ default: &default
   timeout: 5000
 
 development:
-  <<: *default
-  database: db/development.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: cinema_development
+  host: localhost
+  pool: 5
+  username: postgres
+  password: admin
```

Comment:
You should probably move common things to `default` and maybe not hard  code username and password - I mean it would not work with that credentials on my machine, as I have different password.